test/navigation/NavigatorSpec.scala

```diff
@@ -44,9 +44,9 @@ class NavigatorSpec extends SpecBase {
       "for the DateOfArrival page" - {
 
         "when trader is XI" - {
-          "must go to DestinationOfficePage page" in {
+          "must go to AcceptMovement page" in {
             navigator.nextPage(DateOfArrivalPage, NormalMode, emptyUserAnswers.copy(ern = "XI123")) mustBe
-              routes.DestinationOfficeController.onPageLoad("XI123", testArc, NormalMode)
+              routes.AcceptMovementController.onPageLoad("XI123", testArc, NormalMode)
           }
         }
 
```

Comment:
I suppose this test is redundant now as no matter the trader, it goes to AcceptMovement?

Reply:
Yea can remove it